src/docfx/restore/RestoreGit.cs

```diff
@@ -54,9 +54,16 @@ async Task AddWorkTrees()
 
                     await ParallelUtility.ForEach(branches, async branch =>
                     {
+                        // Bilingual repos ({branch}-sxs) only depend on non bilingual branch for git commit history,
```

Comment:
that makes the convention everywhere...

Reply:
how about having a method in `LocalizationConvention.cs` to hide `-sxs`:
```csharp
public static string GetBilingualBranch(string branch) => $"{branch}-sxs";
```
Anywhere we need to access it, just use this one.